modules/configuration/base_config.lua

```diff
@@ -0,0 +1,64 @@
+--- Module which provides ATF base configuration
+--
+-- *Dependencies:* none
+--
+-- *Globals:* none
+-- @copyright [Ford Motor Company](https://smartdevicelink.com/partners/ford/) and [SmartDeviceLink Consortium](https://smartdevicelink.com/consortium/)
+-- @license <https://github.com/smartdevicelink/sdl_core/blob/master/LICENSE>
+
+local config = { }
+--- Flag which defines usage of color for reporting
+config.color = true
+
+--- Define timeout for Heartbeat in msec
+config.heartbeatTimeout = 7000
+--- Define timeout to wait for the events that should not occur
+config.zeroOccurrenceTimeout = 2000
+--- Flag which defines whether ATF checks all validations for particular expectation or just till the first which fails
+config.checkAllValidations = false
+--- Define default version of Ford protocol
+-- 1 - basic
+--
+-- 2 - RPC, encryption
+--
+-- 3 - video/audio streaming, heartbeat
+-- 4 - SDL 4.0
+config.defaultProtocolVersion = 3
+--- Define path to SDL binary
+-- Example: "/home/user/sdl_build/bin"
+config.pathToSDL = ""
+--- Define path to SDL .INI file
+config.pathToSDLConfig = ""
```

Comment:
The description above is somewhat misleading since this variable doesn't contain the path to the `smartDeviceLink.ini`

from `SDL.lua`
```lua
function SDL.INI.file()
  local path = config.pathToSDLConfig
  if path == nil or path == "" then
    path = config.pathToSDL
  end
  return path .. "smartDeviceLink.ini"
end
```
it looks like atf uses (config.pathToSDLConfig + 'smartDeviceLink.ini') as the path to the .INI file.

Also, I think the default value should be changed to
```suggestion
config.pathToSDLConfig = config.pathToSDL
```

Reply:
@ShobhitAd We suppose the path to .INI file and path to SDL binaries may be different. By default, if `pathToSDLConfig` is not defined, path to SDL binaries will be used. 
Please notice both parameters below are empty by default (in `base_config`):
```
config.pathToSDL = ""
config.pathToSDLConfig = ""
```
Non-empty values in `local`, `remote_linux` config files etc. are just defined as an example, that these parameters can be overridden.